androidshared/src/main/java/org/odk/collect/androidshared/ui/FixMismatchThemeProblemsActivity.kt

```diff
@@ -0,0 +1,43 @@
+package org.odk.collect.androidshared.ui
+
+import android.os.Build
+import android.os.Bundle
+import android.webkit.WebView
+import androidx.appcompat.app.AppCompatActivity
+
+/**
+ * When the app's theme doesn't match the system, the first [WebView] created will cause the host
+ * Activity to be recreated with incorrect resource loading (the system ones instead of the custom
+ * set dark/light resources). This doesn't happen for subsequent [WebView] creations however.
+ *
+ * This Activity can be started at launch to be "sacrificed" so no other Activity has to deal
+ * with the problem.
+ *
+ * See [this issue](https://issuetracker.google.com/issues/37124582) for more details.
```

Comment:
Would be good to have a link to our issue too.

Reply:
There wasn't an issue for it I could find! We added it as part of problems found in https://github.com/getodk/collect/pull/4803. There's a lot of discussion in there, so I thought linking to the root cause felt clearer.